lib/datasource/remote/api/region_repository.dart

```diff
@@ -0,0 +1,223 @@
+
+    final request = createRequest(
+      code: SeedsCode.accountRegion,
+      scope: SeedsCode.accountRegion.value,
+      table: SeedsTable.tableRegionMembers,
+      indexPosition: 2,
+      // ignore: avoid_redundant_argument_values
+      keyType: 'i64',
+      lowerBound: region,
+      upperBound: region,
+      limit: 1000,
+    );
+
+    return http
+        .post(membersURL, headers: headers, body: request)
+        .then((http.Response response) => mapHttpResponse<List<RegionMemberModel>>(response, (dynamic body) {
+              final List<dynamic> items = body['rows'].toList();
+              return items.map((item) => RegionMemberModel.fromJson(item)).toList();
+            }))
+        .catchError((error) => mapHttpError(error));
+  }
+
+  ///
+  /// Create Region
+  ///
+  /// This action will transfer 1,000 Seeds region fee, then create a region with the name
+  /// regionAccount must end in ".rgn", e.g. bali.rgn and it must be a valid Telos account name
+  ///
+  /// Note: The region fee is variable, see getRegionFee().
+  ///
+  Future<Result<TransactionResponse>> create({
+    required String founderAccount,
+    required String regionAccount,
+    required String description,
+    required double latitude,
+    required double longitude,
+    double regionFee = 1000.0,
+  }) async {
+    print('[eos] create region $regionAccount');
+
+    final transaction = buildFreeTransaction([
+      Action()
+        ..account = SeedsCode.accountToken.value
+        ..name = SeedsEosAction.actionNameTransfer.value
+        ..authorization = [
+          Authorization()
+            ..actor = founderAccount
+            ..permission = permissionActive
+        ]
+        ..data = {
+          'from': founderAccount,
+          'to': SeedsCode.accountRegion.value,
+          'quantity': '${regionFee.toStringAsFixed(4)} $currencySeedsCode',
+          'memo': 'Create region fee',
+        },
+      Action()
+        ..account = SeedsCode.accountRegion.value
+        ..name = SeedsEosAction.actionNameCreateRegion.value
+        ..authorization = [
+          Authorization()
+            ..actor = founderAccount
+            ..permission = permissionActive
+        ]
+        ..data = {
+          'founder': founderAccount,
+          'rgnaccount': regionAccount,
+          'description': description,
+          'locationJson': '{lat:$latitude,lon:$longitude}',
+          'latitude': latitude,
+          'longitude': longitude
+        },
+    ], founderAccount);
+
+    return buildEosClient()
+        .pushTransaction(transaction)
+        .then((dynamic response) => mapEosResponse<TransactionResponse>(response, (dynamic map) {
+              return TransactionResponse.fromJson(map);
+            }))
+        .catchError((error) => mapEosError(error));
+  }
+
+  ///
+  /// Join a region
+  ///
+  Future<Result<TransactionResponse>> join({
+    required String region,
+    required String account,
```

Comment:
joiningUserAccount

Reply:
we generally call the user account... userAccount I think

I just made these mimic the on chain calls. 

Not sure what's a better approach.